src/TurbulenceClosures/turbulence_closure_implementations/TKEBasedVerticalDiffusivities/tke_dissipation_vertical_diffusivity.jl

```diff
@@ -0,0 +1,356 @@
+                                                                       minimum_stratification_number_safety_factor,
+                                                                       negative_tke_damping_time_scale,
+                                                                       tke_dissipation_time_step)
+end
+
+TKEDissipationVerticalDiffusivity(FT::DataType; kw...) =
+    TKEDissipationVerticalDiffusivity(VerticallyImplicitTimeDiscretization(), FT; kw...)
+
+const TDVD{TD} = TKEDissipationVerticalDiffusivity{TD} where TD
+const TDVDArray{TD} = AbstractArray{<:TDVD{TD}} where TD
+const FlavorOfTD{TD} = Union{TDVD{TD}, TDVDArray{TD}} where TD
+
+@inline Base.eltype(::TKEDissipationVerticalDiffusivity{<:Any, <:Any, <:Any, <:Any, FT}) where FT = FT
+
+"""
+    TKEDissipationVerticalDiffusivity([time_discretization = VerticallyImplicitTimeDiscretization(),
+                                      FT = Float64;]
+                                      tke_dissipation_equations = TKEDissipationEquations(),
+                                      stability_functions = VariableStabilityFunctions(),
+                                      minimum_length_scale = StratifiedDisplacementScale(),
+                                      maximum_tracer_diffusivity = Inf,
+                                      maximum_tke_diffusivity = Inf,
+                                      maximum_dissipation_diffusivity = Inf,
+                                      maximum_viscosity = Inf,
+                                      minimum_tke = 1e-6,
+                                      minimum_stratification_number_safety_factor = 0.73,
+                                      negative_tke_damping_time_scale = 1minute,
+                                      tke_dissipation_time_step = nothing)
+
+Return the `TKEDissipationVerticalDiffusivity` turbulence closure for vertical mixing by
+small-scale ocean turbulence based on the prognostic evolution of subgrid
+Turbulent Kinetic Energy (TKE).
+
+!!! note "TKEDissipationVerticalDiffusivity"
+    `TKEDissipationVerticalDiffusivity` is a new turbulence closure diffusivity. The default
+    values for its free parameters are obtained from calibration against large eddy
+    simulations. For more details please refer to [Wagner23catke](@cite).
+
+    Use with caution and report any issues with the physics at https://github.com/CliMA/Oceananigans.jl/issues.
+
+Arguments
+=========
+
+- `time_discretization`: Either `ExplicitTimeDiscretization()` or `VerticallyImplicitTimeDiscretization()`;
+                         default `VerticallyImplicitTimeDiscretization()`.
+
+- `FT`: Float type; default `Float64`.
+
+
+Keyword arguments
+=================
+
+- `maximum_diffusivity`: Maximum value for tracer, momentum, and TKE diffusivities.
+                        Used to clip the diffusivity when/if CATKE predicts
+                        diffusivities that are too large.
+                        Default: `Inf`.
```

Comment:
There is a remnant CATKE reference here!!

```suggestion
                         Used to clip the diffusivity when/if CATKE predicts
                         diffusivities that are too large.
                         Default: `Inf`.
```